helm-chart/renku/templates/ingress.yaml

```diff
@@ -62,6 +62,10 @@ spec:
           serviceName: {{ $gitlabFullname }}
           servicePort: {{ $gitlabServicePort }}
       {{- end }}
+      - path: /repo
```

Comment:
Should we call it `/repos`? We use the plural form in most or all our URLs elsewhere.